.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: 🐛 Bug Report
+about: If something isn't documented correctly 🤔
+
+---
+
+Preferred is to submit a suggestion via pull request: For small changes the
```

Comment:
This is also a suggestion:
I would comment out the "hint texts" in general. Or at least the long text at the beginning if the comment formatting is too distracting.

Otherwise it can be distracting and bloats up the issue if the people leave them in  (and not everyone will remove them, I would bet on that).

That also makes it clearer (I hope) what should be replaced and would should be left untouched (e.g. the headers). 

The downside is when writing the issue you have formatting the user might not be familiar with. 

e.g. use

```
<!-- hint (can be removed)
Prererred is to submit ...
--> 
```

```
<!-- hint (can be removed)
A clear and concise description of what the problem is.
--> 
```

short form also possible:

```
<!-- A clear and concise description of what the problem is. --> 
```

Reply:
I leave out comments for now. THis is just a template (suggestion) to guide a user, one can delete the whole template and write what they want. If comment or not, the text can be removed. This is also intended as there are cases where not all sections are applicable (that's also why not using fields explicitly).